utilities/ideascale-importer/ideascale_importer/reviews_importer.py

```diff
@@ -0,0 +1,102 @@
+from lxml import html
+import uuid
+
+from ideascale_importer import utils
+
+def random_string(string_length=6):
```

Comment:
A suggestion to do the same in one line:

```
import secrets

....
def random_string(string_length=6):
    return secrets.token_hex(string_length)
```

Reply:
done